allegro.h

```diff
@@ -682,7 +692,11 @@ typedef class Alg_track : public Alg_event_list {
     Alg_track(Alg_event_list_ref event_list, Alg_time_map_ptr map, 
               bool units_are_seconds);
     virtual ~Alg_track() { // note: do not call set_time_map(NULL)!
-        if (time_map) time_map->dereference(); time_map = NULL; }
+        if (time_map) {
+            time_map->dereference();
+        }
+        time_map = NULL;
```

Comment:
On multiple lines, it would be clearer to put time_map = NULL into the braces so it is associated with dereference(). The idea is that we're removing a reference to time_map and decrementing the refcount. Maybe this should have been a macro like UNREF(time_map) to dereference AND set to NULL, but at least the operations should be seen as a pair.

Reply:
In the original code, `time_map` will be set to `NULL` regardless of if the `if` statement runs or not:

```c++
if (time_map) time_map->dereference(); time_map = NULL; }
```

> the operations should be seen as a pair
In that case, it looks like the original line was incorrect. I'll go ahead and make that change.